### PR TITLE
HL-23

### PR DESCRIPTION
**Pull request by pondpiu**
[Link to ticket](https://dltx.atlassian.net/jira/software/projects/HL/boards/82?selectedIssue=HL-23)

- adjust text display for each transaction type

image

### CODE REVIEW

File: src/types/entities.ts [hunk 1]
```diff
@@ -12,8 +12,9 @@ export type VaultTransaction = {
 
 export type FormattedVaultTransaction = Pick<
   VaultTransaction,
-  "id" | "vaultAddress" | "depositerAddress" | "type"
+  "id" | "vaultAddress" | "depositerAddress"
 > & {
+  type: string;
```
Comment (brettvniekerk) [inline src/types/entities.ts:17]: should this not also be `withdraw | deposit` ?

Comment (pondpiu) [reply src/types/entities.ts:17]: I make the type looser to allow mapping string into formatted value.
Now that you mention it, i prob. better to leave it strict and map it right on the View component

**Review by bitcoinbrisbane — APPROVED** [review]

**Review by JasminFarhi — APPROVED** [review]

File: src/types/entities.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { Address } from "wagmi";
 
 export type VaultTransaction = {
   id: Address;
-  type: string;
+  type: "withdraw" | "deposit";
```
Comment (bazmatic) [inline src/types/entities.ts:6]: I suggest using enums if you're going to do this

**Review by OptimusOpus — APPROVED** [review]
LGTM

**Review by brettvniekerk — APPROVED** [review]

**Review by JasminFarhi — APPROVED** [review]

**Pull request merged**